Source/WebCore/rendering/RenderTableCol.cpp

```diff
@@ -168,21 +172,22 @@ void RenderTableCol::clearPreferredLogicalWidthsDirtyBits()
 {
     setPreferredLogicalWidthsDirty(false);
 
-    for (auto& child : childrenOfType<RenderObject>(*this))
-        child.setPreferredLogicalWidthsDirty(false);
+    for (CheckedRef<RenderObject> child : childrenOfType<RenderObject>(*this))
+        child->setPreferredLogicalWidthsDirty(false);
 }
 
-RenderTable* RenderTableCol::table() const
+CheckedPtr<RenderTable> RenderTableCol::table() const
 {
-    auto table = parent();
+    CheckedPtr table = parent();
     if (table && !is<RenderTable>(*table))
         table = table->parent();
+
     return dynamicDowncast<RenderTable>(table);
 }
 
-RenderTableCol* RenderTableCol::enclosingColumnGroup() const
+CheckedPtr<RenderTableCol> RenderTableCol::enclosingColumnGroup() const
```

Comment:
I think all of these changes are against the style guide. These should have a separate `checked` method.

cc @cdumez